tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -145,6 +146,7 @@ func (ls *leaseStresser) Stress() error {
 
 func (ls *leaseStresser) run() {
 	defer ls.runWg.Done()
+	ls.keyOps = 0
```

Comment:
my original though was that `keyOps` only records number of operation per test case. the tester accumulate that number for the entire round. that's why i reset it every test case. 

an alternate approach is that, we never reset `keyOps` and let it accumulate. 
then we let tester to figure the total keyOps for a round by find the difference from `KeyOps` of last round and current `keyOps`.